### ISSUE TITLE
Windows key opens windows start menu

### ISSUE THREAD

**Issue by All-Ki** [cairoshell/cairoshell#128]
steps : Press the windows key.
Expected : 
Apps menu item opens
Result : 
Apps menu item flickers, windows start menu opens

**Comment by josuave** [comment]
I added this in my fork. https://github.com/josuave/cairoshell

**Comment by dremin** [comment]
@josuave Might need to look at this some more since it breaks winkey hotkey combos. I tried some code changes so that it only invokes on keyup when no other keys were pressed, but then other apps think the key is still in the pressed state. Not sure if there is a way to have both work.

**Comment by josuave** [comment]
Ya, I noticed that too. There was another implementation for a keyboard hook that I saw a while back. I remember it handling modifier keys in a more flexible manner.

Sent from my iPhone

On Apr 22, 2018, at 3:03 PM, Sam Johnson <notifications@github.com<mailto:notifications@github.com>> wrote:


@josuave<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjosuave&data=02%7C01%7C%7Ced66c8cea8124e9b13c708d5a883bf05%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636600206148418705&sdata=%2FMYVESiXJJEuY9yTYKG1LAUxkaY0s%2BjcbvtkD8%2Fk3j0%3D&reserved=0> Might need to look at this some more since it breaks winkey hotkey combos. I tried some code changes so that it only invokes on keyup when no other keys were pressed, but then other apps think the key is still in the pressed state. Not sure if there is a way to have both work.
